sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -56,106 +164,477 @@ export interface BrowserPushChannel {
+    getNotificationHubJob(jobId: string, options?: OperationOptions): Promise<NotificationHubJob>;
+    getNotificationOutcomeDetails(notificationId: string, options?: OperationOptions): Promise<NotificationDetails>;
+    getRegistration(registrationId: string, options?: OperationOptions): Promise<RegistrationDescription>;
+    listNotificationHubJobs(options?: OperationOptions): Promise<NotificationHubJob[]>;
+    listRegistrations(options?: RegistrationQueryOptions): PagedAsyncIterableIterator<RegistrationDescription>;
+    listRegistrationsByTag(tag: string, options?: RegistrationQueryLimitOptions): PagedAsyncIterableIterator<RegistrationDescription>;
+    scheduleBroadcastNotification(scheduledTime: Date, notification: Notification, options?: OperationOptions): Promise<NotificationHubsMessageResponse>;
+    scheduleNotification(scheduledTime: Date, tags: string[] | string, notification: Notification, options?: OperationOptions): Promise<NotificationHubsMessageResponse>;
+    sendBroadcastNotification(notification: Notification, options?: SendOperationOptions): Promise<NotificationHubsMessageResponse>;
+    sendDirectNotification(pushHandle: PushHandle, notification: Notification, options?: SendOperationOptions): Promise<NotificationHubsMessageResponse>;
+    sendNotification(tags: string[] | string, notification: Notification, options?: SendOperationOptions): Promise<NotificationHubsMessageResponse>;
+    submitNotificationHubJob(job: NotificationHubJob, options?: OperationOptions): Promise<NotificationHubJob>;
+    updateInstallation(installationId: string, patches: JsonPatch[], options?: OperationOptions): Promise<NotificationHubsResponse>;
+    updateRegistration(registration: RegistrationDescription, options?: OperationOptions): Promise<RegistrationDescription>;
 }
 
 // @public
-export interface NotificationHubsClientOptions extends CommonClientOptions {
+export interface NotificationOutcomeCollectionItem {
+    count: number;
+    state: string;
 }
 
+// @public
+export type NotificationOutcomeState = "Enqueued" | "DetailedStateAvailable" | "Processing" | "Completed" | "Abandoned" | "Unknown" | "NoTargetFound" | "Cancelled";
+
 // @public
 export type PushHandle = BrowserPushChannel | string;
 
+// @public
+export type RegistrationDescription = AdmRegistrationDescription | AdmTemplateRegistrationDescription | AppleRegistrationDescription | AppleTemplateRegistrationDescription | BaiduRegistrationDescription | BaiduTemplateRegistrationDescription | BrowserRegistrationDescription | BrowserTemplateRegistrationDescription | GcmRegistrationDescription | GcmTemplateRegistrationDescription | FcmRegistrationDescription | FcmTemplateRegistrationDescription | MpnsRegistrationDescription | MpnsTemplateRegistrationDescription | WindowsRegistrationDescription | WindowsTemplateRegistrationDescription;
+
+// @public
+export interface RegistrationDescriptionCommon {
+    etag?: string;
+    expirationTime?: Date;
```

Comment:
Can we name this `expiresOn`?

Also, I see that this is a `Date` and expirationTime in InstallationCommon is a string, would it make sense to make them consistent?

Reply:
This is a direct mapping to the underlying service ExpirationTime, so keeping consistent with the installation expirationTime and Registration ExpirationTime